src/pages/instances/InstanceDetailHeader.tsx

```diff
@@ -41,6 +43,7 @@ const InstanceDetailHeader: FC<Props> = ({
   const controllerState = useState<AbortController | null>(null);
   const { data: settings } = useSettings();
   const isClustered = isClusteredServer(settings);
+  const isMediumScreen = () => isWidthBelow(820);
```

Comment:
This is not re-run when resizing the window. We might need a local state variable `isMidumScreen` here and add a `useEventListener("resize",...)`, that recalculates the state on the resize event to make it responsive.

Reply:
Oop, I forgot to implement this in the recent push. Please hold.

Reply:
All done!